src/search/YesNoDialog.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { Action } from "../core/Action";
+import { Search } from "./Search";
+import { MatchingStrategy } from "../MatchingStrategy";
+
+interface Props {
+    initialQuery?: string,
```

Comment:
jaki jest w sumie cel initialQuery? nie widzę wykorzystania w kodzie - zawsze jest przekazywany  pusty string

Reply:
To dlatego, że ten kod jest daleki od docelowego ;) 
Rolą initial query jest wstępne ustawienie wartości inputa. Np. w przypadku wyszukiwania notatki zachowanie ma być takie, że nazwa aktywnej notatki jest uzupełniana jako inicjalna.